libs/vkd3d/device.c

```diff
@@ -478,6 +478,7 @@ static const struct vkd3d_debug_option vkd3d_config_options[] =
     {"debug_utils", VKD3D_CONFIG_FLAG_DEBUG_UTILS},
     {"force_static_cbv", VKD3D_CONFIG_FLAG_FORCE_STATIC_CBV},
     {"dxr", VKD3D_CONFIG_FLAG_DXR},
+    {"multi_queue", VKD3D_CONFIG_FLAG_MULTI_QUEUE},
```

Comment:
Might be a good idea to document that in the README.

Reply:
Yes, but can wait until it's more complete.